refinery/solr/data_set_manager/conf/schema.xml

```diff
@@ -156,6 +156,8 @@
 
     <field name="study_uuid" type="text_en" indexed="true" stored="true" multiValued="false" />
 
+    <field name="data_set_uuid" type="text_en" indexed="true" stored="true" multiValued="false" />
```

Comment:
Just take note that we'll need to run the `rebuild_index` command after this change: https://github.com/refinery-platform/refinery-platform/wiki/Solr-development#updated-index-model